api/app/controllers/spree/api/checkouts_controller.rb

```diff
@@ -84,6 +77,33 @@ def user_id
           params[:order][:user_id] if params[:order]
         end
 
+        def update_params
+          if update_params = massaged_params[:order]
```

Comment:
I don't like setting variables in conditional statements. I propose 2 different wat how to handle it:

~~~
return {} if masseged_params[:order].blank?
massaged_params[:order].permit(permitted_checkout_attributes)
~~~

~~~
update_params = massaged_params.fetch(:order, {})
update_params.permit(permitted_checkout_attributes)
~~~